build-plugin/action.yml

```
@@ -99,6 +118,15 @@ runs:
       with:
         subject-path: ${{ steps.package-plugin.outputs.archive }}
 
+    - name: Commit updated changelog
+      if: ${{ inputs.use_changelog_generator == 'true' }}
+      uses: EndBug/add-and-commit@v9
```

All third-party actions should be pinned to their exact commit for security reasons:

```suggestion
      uses: EndBug/add-and-commit@a94899bca583c204427a224a7af87c02f9b325d5 # v9.1.4
```

---

do we really need this action here though? You can easily commit and push by using CLI commands here. e.g.: https://github.com/grafana/plugin-validator/blob/main/.github/workflows/do-release.yml#L31

---

I don't have strong opinion here, it just seemed easier for me. If you think we should use CLI commands, I can do it